app/javascript/js/controllers/table_row_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  visitRecord(event) {
+    if (event.type === 'click') {
+      const isLinkOrButton = event.target.closest('a, button, svg')
```

Comment:
We need to see how the support is for `svg`s that need to be clicked (status field x icon).